src/util/Config.ts

```diff
@@ -0,0 +1 @@
+export const BACKEND_URL = process.env.REACT_APP_BACKEND_URL ?? 'https://617d3fe41eadc50017136443.mockapi.io';
```

Comment:
What is `mockapi.io` ?

Reply:
Free page where you can define simple rest api.